modules/tinymce/src/themes/silver/main/ts/ui/menus/item/build/CardMenuItem.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { AlloyComponent, AlloySpec, Behaviour, Disabling } from '@ephox/alloy';
+import { Menu } from '@ephox/bridge';
+import { Arr, Optional } from '@ephox/katamari';
+import { UiFactoryBackstageShared } from 'tinymce/themes/silver/backstage/Backstage';
+import { renderItemDomStructure } from 'tinymce/themes/silver/ui/menus/item/structure/ItemStructure';
+import ItemResponse from '../ItemResponse';
+import { renderContainer, renderDescription, renderHtml, renderImage } from '../structure/ItemSlices';
+import { replaceText } from './AutocompleteMenuItem';
+import { buildData, renderCommonItem } from './CommonMenuItem';
+
+export interface CardExtras {
+  itemBehaviours?: Array<Behaviour.NamedConfiguredBehaviour<Behaviour.BehaviourConfigSpec, Behaviour.BehaviourConfigDetail>>;
+  title: { // Extras specific to title components
+    matchText?: string;
+    highlight: string[];
+  };
+}
+
+const render = (items: Menu.ContainerItem[], extras: CardExtras): Array<AlloySpec> => Arr.map(items, (item) => {
+  switch (item.type) {
+    case 'cardcontainer':
+      return renderContainer(render(item.items, extras), item.direction);
+
+    case 'image':
+      return renderImage(item.src, item.classes, item.alt);
+
+    case 'description':
+      return renderDescription(item.text, item.classes);
+
+    case 'title':
+      // Only highlight targeted titles
+      const shouldHighlight = item.name.exists((name) => Arr.contains(extras.title.highlight, name));
```

Comment:
This appears to be missing from the `description`?  When we discussed on Thursday last week it was my understanding that should also be able to be highlighted if needed.

Reply:
I'll remove the description component, this [comment](https://github.com/tinymce/tinymce/pull/6149#discussion_r515174905)

Reply:
Sounds good, I assume you'll just go with a `cardtext` or something component instead?

Reply:
Yeah I thought about what we could call the `title` component instead and `cardtext` sounds good and generic